src/pybind/mgr/dashboard/frontend/src/app/shared/components/critical-confirmation-modal/critical-confirmation-modal.component.html

```diff
@@ -10,6 +10,12 @@
           [formGroup]="deletionForm"
           novalidate>
       <div class="modal-body">
+        <cd-alert-panel *ngIf="showMultiClusterMessage"
+                        type="info"
+                        spacingClass="mb-3"
+                        i18n>
+          <p>Please note that the data for the disconnected cluster will remain accessible for a duration of around 5 minutes. After this period, it will be automatically removed.</p>
+        </cd-alert-panel>
```

Comment:
Since this is a shared component, lets keep it generic
```suggestion
        <cd-alert-panel *ngIf="showInfoMessage"
                        type="info"
                        spacingClass="mb-3"
                        i18n>
          <p>{{infoMessage}}</p>
        </cd-alert-panel>
```

Reply:
Thanks, updated